### PR TITLE
provide multi-arch (amd64/arm64) docker images for the router

### PR DESCRIPTION
**Pull request by garypen**
fixes: #1932
fixes: #2098 

The next time we release docker images, they will be multi-arch.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -486,16 +486,17 @@ jobs:
...
             ROUTER_TAG=ghcr.io/apollographql/router
-            # Build debug image
-            docker build --build-arg ROUTER_RELEASE=${VERSION} -f dockerfiles/Dockerfile.router -t ${ROUTER_TAG}:${VERSION}-debug .
-            # Build release image
-            docker build --build-arg ROUTER_RELEASE=${VERSION} -f dockerfiles/Dockerfile.router -t ${ROUTER_TAG}:${VERSION} .
+            # Create a multi-arch builder which works properly under qemu
+            docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+            docker context create buildx-build
+            docker buildx create --driver docker-container --use buildx-build
+            docker buildx inspect --bootstrap
```
Comment (abernix) [inline .circleci/config.yml:493]: What does this line do?  Is it just output?

File: NEXT_CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,12 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 ## ❗ BREAKING ❗
 ## 🚀 Features
 
+### Provide multi-arch (amd64/arm64) docker images for the router ([Issue #1932](https://github.com/apollographql/router/pull/2138))
+
+From the next release, our docker images will be multi-arch.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/2138
```
Comment (abernix) [inline NEXT_CHANGELOG.md:34]: I think we should capitalize things according to their official productized names in the changelog.

```suggestion
### Provide multi-arch (amd64/arm64) Docker images for the Router ([Issue #1932](https://github.com/apollographql/router/pull/2138))

From the next release, our Docker images will be multi-arch.

By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/2138
```

**Review by abernix — APPROVED** [review]
Nice.  One question, one CL suggestion.

File: .circleci/config.yml (hunk 1 above)
Comment (garypen) [reply .circleci/config.yml:493]: It is. I could remove it, but it contains useful debugging information if we have a proble.

Comment (abernix) [reply .circleci/config.yml:493]: works for me!

**Pull request merged**